include/core/LogMacros.h

```diff
@@ -76,16 +105,19 @@
 #ifdef LOG_ABORT
 #undef LOG_ABORT
 #endif
-#define LOG_ABORT(message)                                                     \
-    LOG4CXX_FATAL(ml::core::CLogger::instance().logger(), "" message);         \
+#define LOG_ABORT(message)                                                                   \
+    BOOST_LOG_STREAM_SEV(ml::core::CLogger::instance().logger(), ml::core::CLogger::E_Fatal) \
+    LOG_LOCATION_INFO                                                                        \
+    message;                                                                                 \
     ml::core::CLogger::fatal()
 
 // Log at a level specified at runtime as a string, for example
-// LOG_AT_LEVEL("WARN", << "Stay away from here " << username)
+// LOG_AT_LEVEL(ml::core::CLogger::E_Warn, << "Stay away from here " << username)
```

Comment:
nit: this comment requires updating, now the level is specified as an enum, not a string